### ISSUE TITLE
Missing 'office' for Rajesh Sonkar

### ISSUE THREAD

**Issue by tmtmtmtm** [spencermountain/wtf_wikipedia#530]
With version 10.1.4, when I run `wtf_wikipedia Rajesh Sonkar` it's failing to pick up the "office" line from the infobox:

```
| name                = Dr. Rajesh Sonkar
| image               = 
| birth_date          = {{Birth date and age|df=y|1968|12|9}}
| birth_place         = [[Indore]], [[Madhya Pradesh]], [[India]]
| office              = [[President Bhartiya Janta Party(BJP) Indore, Madhya Pradesh]]
| term_start          = 10 May 2020
| term_end            = 
| constituency        = 
| succeeded           = 
| party               = [[Bharatiya Janata Party]]
```

→ 

```json
        {
          "name": {
            "text": "Dr. Rajesh Sonkar"
          },
          "birth_date": {
            "text": "December 9, 1968"
          },
          "birth_place": {
            "text": "Indore, Madhya Pradesh, India",
            "links": [
              {
                "type": "internal",
                "page": "Indore"
              },
              {
                "type": "internal",
                "page": "Madhya Pradesh"
              },
              {
                "type": "internal",
                "page": "India"
              }
            ]
          },
          "term_start": {
            "text": "10 May 2020"
          },
          "party": {
            "text": "Bharatiya Janata Party",
            "links": [
              {
                "type": "internal",
                "page": "Bharatiya Janata Party"
              }
            ]
          },
```

**Comment by spencermountain** [comment]
oh my gosh, good catch.
will add a fix to the next release
thanks

**Comment by spencermountain** [comment]
fixed in 10.1.5
